Scripts/package.py

```diff
@@ -226,7 +226,7 @@ def path_leaf(path):
     return tail or ntpath.basename(head)
 
 def pathsInDirectory(directory):
-    return list(map(lambda file: path.join(directory, file), os.listdir(directory)))
+    return [path.join(directory, file) for file in os.listdir(directory) if not file.endswith(".pdb")]
```

Comment:
😍 much nicer to look at

Reply:
But I'd argue that this isn't the right place to do the filtering, since this function is called "pathsInDirectory".  How about adding a `if endswith(".pdf"): continue` in the loop that uses this function, instead?